transport/conn.go

```diff
@@ -243,10 +258,9 @@ type Conn struct {
 }
 
 type ConnConfig struct {
-	TCPNoDelay bool
-	Timeout    time.Duration
-	// This will be used.
-	// DefaultConsistency frame.Consistency
+	TCPNoDelay         bool
+	Timeout            time.Duration
+	DefaultConsistency frame.Consistency
```

Comment:
I don't see it being used in our program and linter is ok with it? Is it intentional? @Choraden

Reply:
It's because the fields are exported as well as the structure itself.